lib/ProkkaAnnotation/ProkkaAnnotationImpl.py

```diff
@@ -8,202 +8,213 @@
 from Bio import SeqIO
 from Bio.SeqRecord import SeqRecord
 from BCBio import GFF
-from pprint import pformat
+from pprint import pformat,pprint
+from subprocess import check_output, CalledProcessError
+import sys
 
 from AssemblyUtil.AssemblyUtilClient import AssemblyUtil
 from GenomeFileUtil.GenomeFileUtilClient import GenomeFileUtil
 from KBaseReport.KBaseReportClient import KBaseReport
 from biokbase.workspace.client import Workspace as workspaceService  # @UnresolvedImport @IgnorePep8
+from DataFileUtil.DataFileUtilClient import DataFileUtil
+from collections import namedtuple,defaultdict
 #END_HEADER
 
 
 class ProkkaAnnotation:
-    '''
+    """
     Module Name:
     ProkkaAnnotation
 
     Module Description:
     A KBase module: ProkkaAnnotation
-    '''
+    """
 
     ######## WARNING FOR GEVENT USERS ####### noqa
     # Since asynchronous IO can lead to methods - even the same method -
     # interrupting each other, you must be *very* careful when using global
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "0.0.3"
-    GIT_URL = "https://github.com/kbaseapps/ProkkaAnnotation"
-    GIT_COMMIT_HASH = "caaa3552b4ac0cfd9a172938ff1af663b200cdeb"
+    VERSION = "1.0.5"
+    GIT_URL = "https://github.com/bio-boris/ProkkaAnnotation.git"
+    GIT_COMMIT_HASH = "8c6463159e9effb84f719f124e9d82685cb70c9f"
 
     #BEGIN_CLASS_HEADER
-    def _get_input_value(self, params, key):
+
+    @staticmethod
+    def _get_input_value(params, key):
+        """Get value of key after checking for its existence
```

Comment:
You also might consider moving these methods to another utils file. it's generally better to keep the Impl file relatively clean because many modules have multiple methods